pkgs/applications/misc/swaysettings/default.nix

```
@@ -0,0 +1,78 @@
+{ lib
+, fetchFromGitHub
+, accountsservice
+, appstream-glib
+, dbus
+, desktop-file-utils
+, gettext
+, glib
+, gobject-introspection
+, gsettings-desktop-schemas
+, gtk3
+, json-glib
+, libgee
+, libhandy
+, libxml2
+, meson
+, ninja
+, pantheon
+, pkg-config
+, python3Full
```

```suggestion

, python3
```

As I have said, pythonFull is not to be used here.

---

corrected